packages/synapse-client/src/SynapseClient.ts

```diff
@@ -96,15 +135,13 @@ export class SynapseClient extends BaseAPI {
   public wikiPageServices2Client: WikiPageServices2Api
   public wikiPageServicesClient: WikiPageServicesApi
 
-  constructor(protected configuration = DEFAULT_CONFIG) {
+  constructor(configurationParameters?: ConfigurationParameters) {
```

Comment:
Update the constructor to take `ConfigurationParameters`, which is easy to merge with defaults, unlike a `Configuration` class instance